app/Http/Controllers/VolunteerController.php

```diff
@@ -105,4 +105,13 @@ public function storeFeedback(Request $request)
         \Session::flash('flash_message','feedback successfully sent!');
         return redirect('/');
     }
+    /**
+     * Show all my events
+     */
+    public function showEvents()
+    {
+        $user = Auth::user();
+        $followedEvents = $user->events();
```

Comment:
events() returns all events that this user either is following or registered for. You would better call the variable **allEvents** and the function showAllEvents in case of further filteration